contributor_docs/unit_testing.md

```diff
@@ -249,75 +249,3 @@ visualSuite('3D Model rendering', function() {
 });
 ```
 
-
-Different operating systems and browsers render graphics with subtle variations. These differences are normal and shouldn't cause tests to fail.
```

Comment:
@davepagurek This is an experimental PR, critique welcome, I was trying to think of an efficient way to keep the non-code parts of the branches in sync. It raises the question: assuming this is only applicable in 2.0 (is that the case, actually?), does it make sense to add it to the main doc on unit testing but with a note that it applies to dev-2.0 only?

Reply:
We have snapshot tests on both, but we've updated how the system works in 2.0 and used a new diffing algorithm now that we have more tests. So, adding it to main makes sense, and we can mention that the algorithm described is for 2.0, and 1.x uses something similar but simpler and less robust.